### PR TITLE
Update throttling-limits.md

### PR DESCRIPTION
**Pull request by keylimesoda**
Small update to throttling limits.

### CODE REVIEW

File: concepts/throttling-limits.md [hunk 1]
```diff
@@ -27,11 +27,12 @@ Any request can be evaluated against multiple limits, depending on the scope of
 
 The following limits apply to requests on the assignment service API:
 
-| Request Type                 | Limit per app per tenant     | Limit per tenant for all apps |
+| Request type                 | Limit per app per tenant     | Limit per tenant for all apps |
 |---------------------------|------------------------------|----------------------------|
 | Any         | 500 requests per 10 seconds   | 1000 requests per 10 seconds
 |Any          | 15,000 requests per 3600 seconds|30,000 requests per 3600 seconds|
 | GET me/Assignment  | 50 requests per 10 seconds | 150 requests per 10 seconds |
+| POST /reauthorize subscription by ID  |  4000 requests per 20 seconds  |  1000 requests per 20 seconds
```
Comment (FaithOmbongi) [inline concepts/throttling-limits.md:35]: Hi @keylimesoda - For subscription-related metrics, update the [Subscription service limits](https://learn.microsoft.com/en-us/graph/throttling-limits#subscription-service-limits) section

**Review by FaithOmbongi — CHANGES_REQUESTED** [review]
Hi @keylimesoda - For subscription-related metrics, update the [Subscription service limits](https://learn.microsoft.com/en-us/graph/throttling-limits#subscription-service-limits) section

File: concepts/throttling-limits.md (hunk 1 above)
Comment (keylimesoda) [reply concepts/throttling-limits.md]: @FaithOmbongi please review updated changes.

**Comment by keylimesoda** [comment]
@FaithOmbongi Please review changes.

**Review by FaithOmbongi — APPROVED** [review]

**Pull request merged**